launcher

```diff
@@ -693,7 +693,7 @@ run_bootstrap() {
   declare -i BOOTSTRAP_EXITCODE
   rm -f $cidbootstrap
 
-  echo "$input" | $docker_path run --shm-size=512m $user_args $links "${env[@]}" -e DOCKER_HOST_IP="$docker_ip" --cidfile "$cidbootstrap" -i -a stdin -a stdout -a stderr $volumes $image \
+  echo "$input" | $docker_path run --shm-size=512m $user_args $links "${env[@]}" -e DOCKER_HOST_IP="$docker_ip" --cidfile "$cidbootstrap" -i -a stdin -a stdout -a stderr -v $PWD:/discourse_docker $volumes $image \
```

Comment:
We can just include the templates under `/etc/nginx/templates/*.template` instead of having to mount a new volume when running the image. The files are not huge so the increase in disk space should not have a significant impact on the overall image size.

Reply:
What do you mean? Have them in the base image or have them inline in `*.template.yml`?

Reply:
Just have them in the base image :)